doc/classes/@GlobalScope.xml

```diff
@@ -2933,7 +2933,10 @@
 		<constant name="PROPERTY_HINT_PASSWORD" value="36" enum="PropertyHint">
 			Hints that a string property is a password, and every character is replaced with the secret character.
 		</constant>
-		<constant name="PROPERTY_HINT_MAX" value="39" enum="PropertyHint">
+		<constant name="PROPERTY_HINT_INPUT_NAME" value="39" enum="PropertyHint">
+			Hints that a string property is an input action name, and can select any action name from the input map. Using the [code]"show_builtin"[/code] hint, built-in input actions can be used. Using the [code]"default_empty"[/code] hint, loose mode is enabled.
```

Comment:
"default_empty" is a really bad name for a feature that allows entering custom values. I would rename it to something like "allow_custom" or similar. Internally Godot calls this `loose_mode`, which would work too.